pom.xml

```diff
@@ -87,6 +87,7 @@
         <module>core/trino-main</module>
         <module>core/trino-parser</module>
         <module>core/trino-server</module>
+        <module>core/trino-server-dev</module>
```

Comment:
is it really `/core`? did you consider `/testing`?

Reply:
Moved